### PR TITLE
Papa 8 cruise deployment/recovery updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
See attached deploy/recovery tables for details.
Glider 276 was not deployed, so I removed the Papa 8 pre-cruise deploy details from the deploy.csv
Glider 363 was deployed then recovered a week later due to a rudder issue (I added a note, but not sure if I should or if that will just be captured in annotation?)
GP03FLMB-8 deployment depth is pretty far off from the design depth (design depth Location B - 4145 m; deployment depth = 4166 m) - discovered that there was an error in the Waypoints table in the cruise plan.  Target longitude should have been 144° 23.874' but was listed as 144° 28.874'.  This might require an annotation?

For recoveries there were a couple oddities:
GP02HYPM-7 recovery, Dee initially had the release 'on deck' times in the table instead of the release fired times.  She updated it, but forgot to fix the date for the anchor release.  I am able to see the elog and verified that the date should be 25 July, not 26 July. 
GP03FLMA-7 - the midwater release was inoperable, so I only used the anchor release time. Per Dee - "The FLMA-7 mid-water release failed to communicate.  Totally dead.  Jim believes an electrical leak from the mooring instruments caused bi-metallic corrosion/electrolysis of the release." 

[QLR tables 29 July_revised_RGTcomments.docx](https://github.com/WHOIGit/ooicgsn-asset-management/files/6902509/QLR.tables.29.July_revised_RGTcomments.docx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
I checked w James this morning and the GP03FLMB-8 30m CTDMO is confirmed at 50m; based on that I updated all the sensor depths (+20m) in the GP03FLMB deploy csv for deployment 8.

File: deployment/GP02HYPM_Deploy.csv [hunk 1]
```diff
@@ -77,31 +77,31 @@ SR1811,,SKQ201920S,,GP02HYPM-WFP03-01-FLORDL000,6,1,2018-07-22T21:51:00,2019-09-
...
+SKQ201920S,,SKQ202111S,,GP02HYPM-WFP02-05-VEL3DL000,7,1,2019-09-26T00:05:00,2021-07-25T20:20:00,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,50.11518,-144.91612,,2095,4237,,,
+SKQ201920S,,SKQ202111S,,GP02HYPM-WFP03-00-WFPENG000,7,1,2019-09-26T00:05:00,2021-07-25T22:22:00,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.11518,-144.91612,,4063,4237,,,
+SKQ201920S,,SKQ202111S,,GP02HYPM-WFP03-01-FLORDL000,7,1,2019-09-26T00:05:00,2021-07-25T22:22:00,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.11518,-144.91612,,4063,4237,,,
+SKQ201920S,,SKQ202111S,,GP02HYPM-WFP03-03-DOSTAL000,7,1,2019-09-26T00:05:00,2021-07-25T22:22:00,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.11518,-144.91612,,4063,4237,,,
+SKQ201920S,,SKQ202111S,,GP02HYPM-WFP03-04-CTDPFL000,7,1,2019-09-26T00:05:00,2021-07-25T22:22:00,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.11518,-144.91612,,4063,4237,,,
+SKQ201920S,,SKQ202111S,,GP02HYPM-WFP03-05-VEL3DL000,7,1,2019-09-26T00:05:00,2021-07-25T22:22:00,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.11518,-144.91612,,4063,4237,,,
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:93]: Recovery times look good.

File: deployment/GP02HYPM_Deploy.csv [hunk 2]
```diff
@@ -77,31 +77,31 @@ SR1811,,SKQ201920S,,GP02HYPM-WFP03-01-FLORDL000,6,1,2018-07-22T21:51:00,2019-09-
...
+SKQ202111S,,,,GP02HYPM-WFP02-04-CTDPFL000,8,1,2021-07-22T01:55:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-CTDPFL-00134,50.069638,-144.800675,,2090,4220,,,
+SKQ202111S,,,,GP02HYPM-WFP02-05-VEL3DL000,8,1,2021-07-22T01:55:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.069638,-144.800675,,2090,4220,,,
+SKQ202111S,,,,GP02HYPM-WFP03-00-WFPENG000,8,1,2021-07-22T01:55:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.069638,-144.800675,,4063,4220,,,
+SKQ202111S,,,,GP02HYPM-WFP03-01-FLORDL000,8,1,2021-07-22T01:55:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.069638,-144.800675,,4063,4220,,,
+SKQ202111S,,,,GP02HYPM-WFP03-03-DOSTAL000,8,1,2021-07-22T01:55:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.069638,-144.800675,,4063,4220,,,
+SKQ202111S,,,,GP02HYPM-WFP03-04-CTDPFL000,8,1,2021-07-22T01:55:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGINS-CTDPFL-00107,50.069638,-144.800675,,4063,4220,,,
+SKQ202111S,,,,GP02HYPM-WFP03-05-VEL3DL000,8,1,2021-07-22T01:55:00,,CGMGP-02HYPM-00008,CGVEH-GWFPGX-93602,CGINS-VEL3DL-01024,50.069638,-144.800675,,4063,4220,,,
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:107]: Deployment times and locations look good.

File: deployment/GP03FLMA_Deploy.csv [hunk 3]
```diff
@@ -107,39 +107,39 @@ SR1811,,SKQ201920S,,GP03FLMA-RIS01-00-SIOENG000,6,1,2018-07-23T22:54:00,2019-09-
...
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOH049,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOH-11683,50.0227,-144.360783,,750,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOH050,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOH-10264,50.0227,-144.360783,,1000,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIM01-02-CTDMOH051,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-CTDMOH-12389,50.0227,-144.360783,,1500,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIS01-00-SIOENG000,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGCON-SIOCON-14876,50.0227,-144.360783,,30,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIS01-03-DOSTAD000,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-DOSTAD-00379,50.0227,-144.360783,,30,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIS01-04-PHSENF000,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-PHSENF-P0103,50.0227,-144.360783,,30,4103,,,
+SKQ201920S,,SKQ202111S,,GP03FLMA-RIS01-05-FLORTD000,7,1,2019-09-29T04:40:00,2021-07-26T18:24:00,CGMGP-03FLMA-00007,,CGINS-FLORTD-01122,50.0227,-144.360783,,30,4103,,,
```
Comment (snwhite) [inline deployment/GP03FLMA_Deploy.csv:127]: Recovery times look good.  All times are anchor release; mid-water release failed to fire.

File: deployment/GP03FLMA_Deploy.csv [hunk 4]
```diff
@@ -107,39 +107,39 @@ SR1811,,SKQ201920S,,GP03FLMA-RIS01-00-SIOENG000,6,1,2018-07-23T22:54:00,2019-09-
...
+SKQ202111S,,,,GP03FLMA-RIM01-02-CTDMOG048,8,1,2021-07-23T19:43:00,,CGMGP-03FLMA-00008,,CGINS-CTDMOG-13598,49.977455,-144.244048,,500,4126,,,
+SKQ202111S,,,,GP03FLMA-RIM01-02-CTDMOH049,8,1,2021-07-23T19:43:00,,CGMGP-03FLMA-00008,,CGINS-CTDMOH-13449,49.977455,-144.244048,,750,4126,,,
+SKQ202111S,,,,GP03FLMA-RIM01-02-CTDMOH050,8,1,2021-07-23T19:43:00,,CGMGP-03FLMA-00008,,CGINS-CTDMOH-13447,49.977455,-144.244048,,1000,4126,,,
+SKQ202111S,,,,GP03FLMA-RIM01-02-CTDMOH051,8,1,2021-07-23T19:43:00,,CGMGP-03FLMA-00008,,CGINS-CTDMOH-13448,49.977455,-144.244048,,1500,4126,,,
+SKQ202111S,,,,GP03FLMA-RIS01-00-SIOENG000,8,1,2021-07-23T19:43:00,,CGMGP-03FLMA-00008,,CGCON-SIOCON-13998,49.977455,-144.244048,,30,4126,,,
+SKQ202111S,,,,GP03FLMA-RIS01-03-DOSTAD000,8,1,2021-07-23T19:43:00,,CGMGP-03FLMA-00008,,CGINS-DOSTAD-00377,49.977455,-144.244048,,30,4126,,,
+SKQ202111S,,,,GP03FLMA-RIS01-04-PHSENF000,8,1,2021-07-23T19:43:00,,CGMGP-03FLMA-00008,,CGINS-PHSENF-P0142,49.977455,-144.244048,,30,4126,,,
+SKQ202111S,,,,GP03FLMA-RIS01-05-FLORTD000,8,1,2021-07-23T19:43:00,,CGMGP-03FLMA-00008,,CGINS-FLORTD-01323,49.977455,-144.244048,,30,4126,,,
```
Comment (snwhite) [inline deployment/GP03FLMA_Deploy.csv:145]: Deployment times and locations look good.

File: deployment/GP03FLMB_Deploy.csv [hunk 5]
```diff
@@ -107,39 +107,39 @@ SR1811,,SKQ201920S,,GP03FLMB-RIS01-00-SIOENG000,6,1,2018-07-24T23:01:00,2019-09-
...
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIM01-02-CTDMOH069,7,1,2019-09-27T21:26:00,2021-07-27T19:12:00,CGMGP-03FLMB-00007,,CGINS-CTDMOH-12388,50.37555,-144.5145,,750,4176,,,
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIM01-02-CTDMOH070,7,1,2019-09-27T21:26:00,2021-07-27T19:12:00,CGMGP-03FLMB-00007,,CGINS-CTDMOH-12618,50.37555,-144.5145,,1000,4176,,,
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIM01-02-CTDMOH071,7,1,2019-09-27T21:26:00,2021-07-27T19:12:00,CGMGP-03FLMB-00007,,CGINS-CTDMOH-10228,50.37555,-144.5145,,1500,4176,,,
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIS01-00-SIOENG000,7,1,2019-09-27T21:26:00,2021-07-27T18:00:00,CGMGP-03FLMB-00007,,CGCON-SIOCON-13284,50.37555,-144.5145,,30,4176,,,
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIS01-03-DOSTAD000,7,1,2019-09-27T21:26:00,2021-07-27T18:00:00,CGMGP-03FLMB-00007,,CGINS-DOSTAD-00519,50.37555,-144.5145,,30,4176,,,
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIS01-04-PHSENF000,7,1,2019-09-27T21:26:00,2021-07-27T18:00:00,CGMGP-03FLMB-00007,,CGINS-PHSENF-P0147,50.37555,-144.5145,,30,4176,,,
+SKQ201920S,,SKQ202111S,,GP03FLMB-RIS01-05-FLORTD000,7,1,2019-09-27T21:26:00,2021-07-27T18:00:00,CGMGP-03FLMB-00007,,CGINS-FLORTD-01298,50.37555,-144.5145,,30,4176,,,
```
Comment (snwhite) [inline deployment/GP03FLMB_Deploy.csv:127]: Recovery times look good.

File: deployment/GP03FLMB_Deploy.csv [hunk 6]
```diff
@@ -107,39 +107,39 @@ SR1811,,SKQ201920S,,GP03FLMB-RIS01-00-SIOENG000,6,1,2018-07-24T23:01:00,2019-09-
...
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOG068,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOG-13429,50.331522,-144.480312,,520,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH069,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13631,50.331522,-144.480312,,770,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH070,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-12226,50.331522,-144.480312,,1020,4166,,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH071,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13632,50.331522,-144.480312,,1520,4166,,,
+SKQ202111S,,,,GP03FLMB-RIS01-00-SIOENG000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGCON-SIOCON-14883,50.331522,-144.480312,,50,4166,,,
+SKQ202111S,,,,GP03FLMB-RIS01-03-DOSTAD000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-DOSTAD-00378,50.331522,-144.480312,,50,4166,,,
+SKQ202111S,,,,GP03FLMB-RIS01-04-PHSENF000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-PHSENF-P0180,50.331522,-144.480312,,50,4166,,,
+SKQ202111S,,,,GP03FLMB-RIS01-05-FLORTD000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-FLORTD-01324,50.331522,-144.480312,,50,4166,,,
```
Comment (snwhite) [inline deployment/GP03FLMB_Deploy.csv:145]: Deployment times and locations look good.  Looks like deployment depths were increased by 20 m due to the mooring being deployed 21 m deeper than designed.  We don't usually tweak the design depths, but given the magnitude of the different, this looks good.  I wonder if we should add a note to those that they were "Deployed 20 m deeper than the design depth."

File: deployment/GP05MOAS-GL276_Deploy.csv [hunk 7]
```diff
@@ -7,7 +7,3 @@ SR1811,,TY-1902,,GP05MOAS-GL276-00-ENG000000,2,1,2018-07-23T06:33:00,2019-02-12T
 SR1811,,TY-1902,,GP05MOAS-GL276-01-FLORDM000,2,1,2018-07-23T06:33:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-FLORTM-03260,50.1295,-144.5424,,1000,1000,FLORT deployed in place of customary FLORD,,
 SR1811,,TY-1902,,GP05MOAS-GL276-02-DOSTAM000,2,1,2018-07-23T06:33:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-DOSTAM-00145,50.1295,-144.5424,,1000,1000,,,
 SR1811,,TY-1902,,GP05MOAS-GL276-04-CTDGVM000,2,1,2018-07-23T06:33:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-00104,50.1295,-144.5424,,1000,1000,,,
-SKQ202111S,,,,GP05MOAS-GL276-00-ENG000000,3,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGCON-GLDROO-00276,50.12,-144.54,,1000,1000,,,
-SKQ202111S,,,,GP05MOAS-GL276-01-FLORDM000,3,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-FLORDM-03583,50.12,-144.54,,1000,1000,,,
-SKQ202111S,,,,GP05MOAS-GL276-02-DOSTAM000,3,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-DOSTAM-00398,50.12,-144.54,,1000,1000,,,
-SKQ202111S,,,,GP05MOAS-GL276-04-CTDGVM000,3,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-09238,50.12,-144.54,,1000,1000,,,
```
Comment (snwhite) [inline deployment/GP05MOAS-GL276_Deploy.csv:13]: Looks good.  Not deployed on Papa 8.

File: deployment/GP05MOAS-GL363_Deploy.csv [hunk 8]
```diff
@@ -11,7 +11,7 @@ ZP20200930,,ZP20200930,,GP05MOAS-GL363-00-ENG000000,4,1,2020-10-02T12:59:00,2020
...
-SKQ202111S,,,,GP05MOAS-GL363-02-DOSTAM000,5,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,46.01,-133,,1000,1000,,,
-SKQ202111S,,,,GP05MOAS-GL363-04-CTDGVM000,5,1,2021-07-18T00:00:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-00104,46.01,-133,,1000,1000,,,
+SKQ202111S,,SKQ202111S,,GP05MOAS-GL363-00-ENG000000,5,1,2021-07-21T16:23:00,2021-07-28T16:06:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,50.532413,-144.833735,,1000,1000,rudder issue,,
+SKQ202111S,,SKQ202111S,,GP05MOAS-GL363-01-FLORDM000,5,1,2021-07-21T16:23:00,2021-07-28T16:06:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,50.532413,-144.833735,,1000,1000,rudder issue,,
+SKQ202111S,,SKQ202111S,,GP05MOAS-GL363-02-DOSTAM000,5,1,2021-07-21T16:23:00,2021-07-28T16:06:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,50.532413,-144.833735,,1000,1000,rudder issue,,
+SKQ202111S,,SKQ202111S,,GP05MOAS-GL363-04-CTDGVM000,5,1,2021-07-21T16:23:00,2021-07-28T16:06:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-00104,50.532413,-144.833735,,1000,1000,rudder issue,,
```
Comment (snwhite) [inline deployment/GP05MOAS-GL363_Deploy.csv:17]: Looks good.

File: deployment/GP05MOAS-PG566_Deploy.csv [hunk 9]
```diff
@@ -1,8 +1,8 @@
...
+ZP20200930,,SKQ202111S,,GP05MOAS-PG566-01-CTDGVM000,2,1,2020-10-02T12:45:00,2021-07-21T19:20:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-CTDGVM-09288,46.011,-133.00372,,200,200,,,
+ZP20200930,,SKQ202111S,,GP05MOAS-PG566-02-DOSTAM000,2,1,2020-10-02T12:45:00,2021-07-21T19:20:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-DOSTAM-00537,46.011,-133.00372,,200,200,,,
+ZP20200930,,SKQ202111S,,GP05MOAS-PG566-03-FLORTM000,2,1,2020-10-02T12:45:00,2021-07-21T19:20:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-FLORTM-04039,46.011,-133.00372,,200,200,,,
+ZP20200930,,SKQ202111S,,GP05MOAS-PG566-04-FLORTO000,2,1,2020-10-02T12:45:00,2021-07-21T19:20:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-FLORTO-01333,46.011,-133.00372,,200,200,,,
+ZP20200930,,SKQ202111S,,GP05MOAS-PG566-05-NUTNRM000,2,1,2020-10-02T12:45:00,2021-07-21T19:20:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-NUTNRM-00350,46.011,-133.00372,,200,200,,,
+ZP20200930,,SKQ202111S,,GP05MOAS-PG566-06-PARADM000,2,1,2020-10-02T12:45:00,2021-07-21T19:20:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-PARADM-50201,46.011,-133.00372,,200,200,,,
```
Comment (snwhite) [inline deployment/GP05MOAS-PG566_Deploy.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]
Look good.  One comment that could potentially be addressed.

File: deployment/GP03FLMB_Deploy.csv [hunk 10]
```diff
@@ -107,39 +107,39 @@ SR1811,,SKQ201920S,,GP03FLMB-RIS01-00-SIOENG000,6,1,2018-07-24T23:01:00,2019-09-
...
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH070,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-12226,50.331522,-144.480312,,1020,4166,deployed 20 m deeper than design depth,,
+SKQ202111S,,,,GP03FLMB-RIM01-02-CTDMOH071,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-CTDMOH-13632,50.331522,-144.480312,,1520,4166,deployed 20 m deeper than design depth,,
+SKQ202111S,,,,GP03FLMB-RIS01-00-SIOENG000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGCON-SIOCON-14883,50.331522,-144.480312,,50,4166,deployed 20 m deeper than design depth,,
+SKQ202111S,,,,GP03FLMB-RIS01-03-DOSTAD000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-DOSTAD-00378,50.331522,-144.480312,,50,4166,deployed 20 m deeper than design depth,,
+SKQ202111S,,,,GP03FLMB-RIS01-04-PHSENF000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-PHSENF-P0180,50.331522,-144.480312,,50,4166,deployed 20 m deeper than design depth,,
+SKQ202111S,,,,GP03FLMB-RIS01-05-FLORTD000,8,1,2021-07-24T19:32:00,,CGMGP-03FLMB-00008,,CGINS-FLORTD-01324,50.331522,-144.480312,,50,4166,deployed 20 m deeper than design depth,,
```
Comment (snwhite) [inline deployment/GP03FLMB_Deploy.csv:145]: Deployment time, location and notes look good.

**Review by snwhite — APPROVED** [review]

File: deployment/GP02HYPM_Deploy.csv (hunk 1 above)
Comment (reedan88) [reply deployment/GP02HYPM_Deploy.csv:93]: Looks good.

File: deployment/GP03FLMA_Deploy.csv (hunk 3 above)
Comment (reedan88) [reply deployment/GP03FLMA_Deploy.csv:127]: Looks good.

File: deployment/GP03FLMB_Deploy.csv (hunk 5 above)
Comment (reedan88) [reply deployment/GP03FLMB_Deploy.csv:127]: Looks good.

File: deployment/GP05MOAS-GL276_Deploy.csv (hunk 7 above)
Comment (reedan88) [reply deployment/GP05MOAS-GL276_Deploy.csv:13]: Looks good.

File: deployment/GP05MOAS-GL363_Deploy.csv (hunk 8 above)
Comment (reedan88) [reply deployment/GP05MOAS-GL363_Deploy.csv:17]: Looks good.

File: deployment/GP05MOAS-PG566_Deploy.csv (hunk 9 above)
Comment (reedan88) [reply deployment/GP05MOAS-PG566_Deploy.csv:8]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

**Pull request merged**